### DISCUSSION TITLE
Kibana logs not correcting for time

### DISCUSSION THREAD

**Discussion by lmb2100** [Security-Onion-Solutions/securityonion#11028] [General]
Running dist. Deployment, and my logs in kibana are way off, I can only see tests when I go to the 24hr ago when I just ran them. All the machines on the domain are connected to the domain controller as the ntp server and the times are correct. But the timing for the logs in kibana are off. Any suggestions? I thought i had set the ntp for the manager and sensors to the DC but I could've messed it up. Any suggestions?

**Comment by dougburks** [comment]
First, please review the following if you haven't already:
https://docs.securityonion.net/en/2.3/timezones.html
https://docs.securityonion.net/en/2.3/ntp.html

Then try our SOC Dashboards to see if you have the same problem there as in Kibana. If so, you might try setting your timezone manually under Options as described at https://docs.securityonion.net/en/2.3/timezones.html.

If you need further help, please include more information like your local timezone and its UTC offset.
